README.md

```diff
@@ -1,59 +1,102 @@
+- `RUN_DURATION` how long pod-reaper should run before exiting
+- `EXCLUDE_LABEL_KEY` pod metadata label (of key-value pair) that pod-reaper should exclude
+- `EXCLUDE_LABEL_VALUES` comma-separated list of metadata label values (of key-value pair) that pod-reaper should exclude
+
+Additionally, at least one rule must be enabled, or the pod-reaper will error and exit. See the Rules section below for configuring and enabling rules.
+
+Example environment variables:
+```
+# pod-reaper configuration
+NAMESAPCE=test
+POLL_INTERVAL=30s
+RUN_DURATION=15m
+EXCLUDE_LABEL_KEY=pod-reaper
+EXCLUDE_LABEL_VALUES=disabled,false
+
+# enable at least one rule
+CHAOS_CHANCE=.001
+```
+
+
+#### `NAMESPACE`
+Default value: "" (which will look at ALL namespaces)
+
+Controls which kubernetes namespace the pod-reaper is in scope for the pod-reaper. Note that the pod-reaper uses an `InClusterConfig` which makes use of the service account that kubernetes gives to its pods. Only pods (and namespaces) accessible to this service account will be visible to the pod-reaper. 
+
+#### `POLL_INTERVAL`
+Default value: "1m"
+
+Controls how frequently pod-reaper queries kubernetes for pods. The format follows the go-lang `time.duration` format (example: "1h15m30s"). Pod-Reaper will sleep for this duration between polling for pods. 
+
+#### `RUN_DURATION`
+Default value: "0s" (which corresponds to running indefinitely)
+
+Controls the minimum duration that pod-reaper will run before intentionally exiting. The value of "0s" (or anything equivalent such as the empty string) will be interpreted as an indefinite run duration. The format follows the go-lang `time.duration` format (example: "1h15m30s"). Pod-Reaper will finish waiting for, and running another reap cycle if the duration elapses during a poll interval: so there will always be exactly one cycle after the run duration has elapsed. 
+
+#### `EXCLUDE_LABEL_KEY` and `EXCLUDE_LABEL_VALUES`
+These environment variables are used to build a label selector to exclude pods from reaping. The key must be a properly formed kubernetes label key. Values are a comma-separated (without whitespace) list of kubernetes label values. Setting exactly one of the key or values environment variables will result in an error.
+
+A pod will be exclude from the pod-reaper if the pod has a metadata label has a key corresponding to the pod-reaper's exclude label key, and that same metadata label has a value in the pod-reaper's list of excluded label values. This means that exclusion requires both the pod-reaper and pod to be configured in a compatible way.
+
+## Implemented Rules
+
+### Chaos Chance
+Flags a pod for reaping based on a random number generator.
+
+Enabled and configured by setting the environment variable `CHAOS_CHANCE` with a floating point value. A random number generator will generate a value in range `[0,1)` and if the the generated value is below the configured chaos chance, the pod will be flagged for reaping.
```

Comment:
Talk about the `EXCLUDE_LABEL_KEY` and `EXCLUDE_LABEL_VALUES` again. I.e that you can exclude pods from the chaos chance.